Yoast/yoast-components
# 10595 add missing help text to linking suggestions

## Summary

This PR can be summarized in the following changelog entry:

* Added a help text to the Internal linking suggestions. 

## Test instructions

This PR can be tested by following these steps:

* Check out this branch and run `yarn link`.
* Check out wordpress-seo-premium and run `yarn link yoast-components`
* Build all the files and run grunt build in the premium folder. 
* Go to a post, look at the linking suggestions. The copy for the help text should be the same as in the [design example](https://user-images.githubusercontent.com/6073772/43400531-3d53cae0-940e-11e8-9798-3a13ed510f97.png). 

## Quality assurance


Fixes #[10595](https://github.com/Yoast/wordpress-seo/issues/10595)
